### DISCUSSION TITLE
什么时候支持小米手机HyperOS ?

### DISCUSSION THREAD

**Discussion by xiangnanscu** [getlantern/lantern-forum-cn#956] [General]
无法使用, 打开后黑屏

**Comment by MaxMKMax** [comment]
@xiangnanscu 您好. 现在暂时不支持HyperOS. 非常抱歉啊! 但我们支持 Android, IOS.

**Reply by 116426319wzk** [reply]
HyperOS 也是安卓的运行有问题

**Comment by MaxMKMax** [comment]
@116426319wzk @xiangnanscu 你们好. 我会持续的帮你们反馈给技术团队的. 谢谢你们的耐心等待和对Lantern的期待!
